views/audit/big_query_audit_log_v2.sql

```diff
@@ -0,0 +1,759 @@
+  deleteRowCount,
+  outputRowCount,
+  STRUCT(
+    EXTRACT(MINUTE FROM startTime) AS minuteOfDay,
+    EXTRACT(HOUR FROM startTime) AS hourOfDay,
+    EXTRACT(DAYOFWEEK FROM startTime) - 1 AS dayOfWeek,
+    EXTRACT(DAYOFYEAR FROM startTime) AS dayOfYear,
+    EXTRACT(WEEK FROM startTime) AS week,
+    EXTRACT(MONTH FROM startTime) AS month,
+    EXTRACT(QUARTER FROM startTime) AS quarter,
+    EXTRACT(YEAR FROM startTime) AS year
+  ) AS date,
+  createTime,
+  startTime,
+  endTime,
+  runtimeMs,
+  runtimeSecs,
+  tableCopy,
+  /* This code queries data specific to the Copy operation */ 
+  CONCAT(tableCopydataset_id, '.', tableCopytable_id) 
+    AS tableCopyDestinationTableRelativePath,
+  CONCAT(tableCopyproject_id, '.', tableCopydataset_id, '.', tableCopytable_id) 
+    AS tableCopyDestinationTableAbsolutePath,
+  --IF(eventName = "jobChange", 1, 0) AS numCopies,
+  /* This code queries data specific to the Copy operation */ /* The following code queries data specific to the Load operation in BQ */ totalLoadOutputBytes,
+  (totalLoadOutputBytes / 1000000000) AS totalLoadOutputGigabytes,
+  (totalLoadOutputBytes / 1000000000) / 1000 AS totalLoadOutputTerabytes,
+  /* loadConfig STRUCT */
+  STRUCT(
+    loadsourceUris,
+    loadsourceUrisTruncated,
+    loadcreateDisposition,
+    loadwriteDisposition,
+    loadschemaJson,
+    loadschemaJsonTruncated,
+    loadDestinationTable,
+    STRUCT(
+      loadKmskeyName
+    ) AS destinationTableEncryption
+  ) AS loadConfig,
+  IF(eventName = "jobChange", 1, 0) AS numLoads,
+  /* This ends the code snippet that queries columns specific to the Load operation in BQ */ 
+  /* The following code queries data specific to the Extract operation in BQ */ 
+  REGEXP_CONTAINS( jobId, 'beam') AS isBeamJob,
+  /*queryConfig STRUCT */
+  STRUCT(
+    query,
+    queryTruncated,
+    querycreateDisposition,
+    querywriteDisposition,
+    queryschemaJson,
+    queryschemaJsonTruncated,
+    querydestTable_project_id,
+    querydestTable_dataset_id,
+    querydestTable_table_id,
+    queryPriority,
+    querydefaultDataset,
+    STRUCT(
+      queryKmskeyName
+    ) AS destinationTableEncryption,
+    STRUCT(
+      queryName,
+      querysourceUris
+    ) AS tableDefinitions
+  ) AS queryConfig,
+  /* extractConfig STRUCT*/
+  STRUCT(
+    extractdestinationUris,
+    extractdestinationUrisTruncated,
+    STRUCT(
+      extract_projectid,
+      extract_datasetid,
+      extract_tableid,
+      CONCAT(extract_datasetid, '.', extract_tableid) AS relativeTableRef,
+      CONCAT(extract_projectid, '.', extract_datasetid, '.', extract_tableid) AS absoluteTableRef
+    ) AS sourceTable
+  ) AS extractConfig,
+  IF(eventName = "jobChange", 1, 0) AS numExtracts,
+  /* This ends the code snippet that 
+ columns specific to the Extract operation in BQ */ /* The following code queries data specific to the Query operation in BQ */ REGEXP_CONTAINS(
+    jobconfig_query, 'cloudaudit_googleapis_com_data_access_20200303'
```

Comment:
replace "jobconfig_query" with "query" and remove date suffix from cloudaudit_googleapis_com_data_access_20200303